client/src/App.jsx

```diff
@@ -53,6 +54,11 @@ const router = createBrowserRouter([
     element: <NationalityPage />,
     errorElement: <>Error...</>,
   },
+  {
+    path: "/populate-data",
+    element: <PopulateDataFromDBToStore />,
```

Comment:
if this copm is only for testing - remove it in pr (you can ask to show it to me in review, but in the final pr we dont want test components